addons/cargo/stringtable.xml

```diff
@@ -276,6 +276,7 @@
             <Russian>Вкл. проверку размера груза</Russian>
             <Japanese>貨物の大きさを確認のインタラクションを表示</Japanese>
             <Korean>화물 크기 확인 상호작용 표시</Korean>
+            <French>Montrer la taille prise dans la cargaison.</French>
```

Comment:
I personally don't feel this translation is accurate. Here's my suggestion, but feel free to find a better one:
```suggestion
            <French>Montrer l'interaction « Vérifier la taille de la cargaison »</French>
```

Reply:
For the moment, it's the least bad
```suggestion
            <French>Afficher l'interaction de confirmation de la taille de la cargaison</French>
```

I'll certainly come back to this translation.

The players didn't get a good French translation.

I'm a bit disappointed.

Reply:
I'll revisit the translation later.
But I'll put this up for now.